src/message.py

```diff
@@ -24,36 +25,53 @@ def __init__(self, code, body=None):
 
     def send_response(self):
         """Format reponse for in http protocol."""
-        message = '{0} {1}{2}{3}'.format(
+        # message = '{0} {1}{2}{3}'.format(
```

Comment:
Make sure to delete any commented-out code in your final submission. Leaving text comments to explain what is going on is great, but leaving commented-out code lying around is definitely bad practice.